src/plugins/charts/public/services/colors/color_palette.ts

```diff
@@ -49,7 +49,7 @@ const fraction = function (goal: number) {
  * If the number is greater than the length of seed colors available,
  * new colors are generated up to the value of the input number.
  */
-export function createColorPalette(num?: any): string[] {
+export function createColorPalette(num: number): string[] {
```

Comment:
:information_source: I also improved the TypeScript code in that place here and got rid of the `any`.